src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -363,8 +363,8 @@ private static boolean isLargerThanBweThreshold(
         {
             return true;
         }
-        return Math.abs(previousBwe - currentBwe)
-            >= previousBwe * BWE_CHANGE_THRESHOLD_PCT / 100;
+        return previousBwe - currentBwe
```

Comment:
This is really confusing. 
`p - c < -0.15p` is the same as `c - p > 0.15p`. Which is true only if the bwe grew and it grew by more than 15%. Therefore we completely ignore drops in bwe, and we only react once the increase is over 15% (which is the issue the commit message claims to solve).

Can you please normalize the expressions (remove the extra minus signs and use "current - previous" as that's easier to interpret), and add a comment for the intention?

Reply:
i don't know what i've been thinking nor how it worked in my tests